src/banking_stage.rs

```diff
@@ -6,7 +6,7 @@ use crate::entry::Entry;
 use crate::leader_confirmation_service::LeaderConfirmationService;
 use crate::packet::Packets;
 use crate::packet::SharedPackets;
-use crate::poh_recorder::{PohRecorder, PohRecorderError};
+use crate::poh_recorder::{Leader, PohRecorder, PohRecorderError};
```

Comment:
Can you think up a more precise name than Leader? I don't grok what this thing is.

Reply:
WorkingBank?

Reply:
@garious PoHRecoder::Leader is a bank and entry_sender that are expecting ticks and entries.  Banking Stage sends it to PoHService with a min/max height.  Since ticks are always generated, even when there is no leader, the PoHService weights for PoHRecorder to reach the right height and starts feeding the PoHRecorder::Leader.bank ticks when it reaches the min height.